src/main/kotlin/no/nav/syfo/client/pdfgen/PdfGenClient.kt

```diff
@@ -116,6 +120,25 @@ class PdfGenClient(
         const val INNKALLING_PATH = "$API_BASE_PATH/innkalling"
         const val REFERAT_PATH = "$API_BASE_PATH/referat"
 
-        private val log = LoggerFactory.getLogger(PdfGenClient::class.java)
+        val log: Logger = LoggerFactory.getLogger(PdfGenClient::class.java)
+        val objectMapper: ObjectMapper = configuredJacksonMapper()
     }
 }
+
+fun List<DocumentComponentDTO>.sanitizeForPdfGen(): List<DocumentComponentDTO> {
+    val illegalCharacters = listOf('\u0002')
```

Comment:
Gjør det noen forskjell om det er "\u0002" her?

Reply:
Må være '\u0002' for at det skal bli liste med character og vi kan sjekke på det under. Forenklet fiksen nå.